ast/functions.go

```diff
@@ -423,6 +427,8 @@ func (n *AggregateFuncExpr) Update() error {
 		return n.updateMaxMin(false)
 	case AggFuncSum:
 		return n.updateSum()
+	case AggFuncGroupConcat:
```

Comment:
There will be a pr after all the aggregate pr merged to do the clean up works. Or there will be conflicts.